1.data-exploration/sample_info_data_visualization.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python
```

Comment:
please make sure when you `nbconvert` you send it to a folder called `scripts/nbconverted` like you did in 0.data-download.

```bash
jupyter nbconvert --to=script --FilesWriter.build_directory=scripts/nbconverted *.ipynb
```

Reply:
should there be a scripts directory in each module for each modules respective codes?